### PR TITLE
Trackcreadential30

### PR DESCRIPTION
**Pull request by lavanyayezzuness**
Can you please review the code.

### CODE REVIEW

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls [hunk 1]
```diff
@@ -0,0 +1,35 @@
+public class CredentialTrackingLWCAPEX {
```
Comment (nitinsavanurness) [inline force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: Add sharing keyword and comments as per apex docs

Comment (nitinsavanurness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: rename the class to CredentialTrackingController

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+       UserCredentialService ucs = new UserCredentialService();
+       List<User_Credential__c> totalUserCredentials = ucs.getActiveUserCredentials(userId);
+       return totalUserCredentials;  
+   } 
+
+   
+   @AuraEnabled
+   public static Boolean updateUserCredential(String id, string status)
+   {
+       System.debug('NDEBUG Is is' + id);
```
Comment (nitinsavanurness) [inline force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: remove debugs

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+       System.debug('NDEBUG In updateUserCredential');
+        UserCredentialService ucs = new UserCredentialService();
+        List<User_Credential__c> userCredentials = new List<User_Credential__c>{new User_Credential__c(id=id,status__c=status)};
+        List<Database.SaveResult> srList = ucs.updateUserCredentials(userCredentials);
+        
+       System.debug('NDEBUG In updateUserCredential -- 1');
+        Boolean isSuccess= true;
```
Comment (nitinsavanurness) [inline force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: should this be set after checking for errors?

File: force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls [hunk 4]
```diff
@@ -0,0 +1,18 @@
+@isTest
+public class CredentialTrackingLWCAPEX_Test {
```
Comment (nitinsavanurness) [inline force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls]: make test classes and methods private

File: force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        User usrs = TestDataFactory.createUser();
+        TestDataFactory.assignPboAdminPermissions(usrs);
+        System.runAs(usrs){
+        TestDataFactory.createCredentials();
+        List<Credential__c> creds = [Select id from Credential__c limit 1];
+        TestDataFactory.createUserCredentials(usrs.Id,creds[0].Id);
+        User_Credential__c uc= TestDataFactory.createUserCredentials(usrs.id,creds[0].Id);
+        insert uc;
+        List<User_Credential__c> ucList = CredentialTrackingLWCAPEX.getUserCredentials(usrs.id);
+        CredentialTrackingLWCAPEX.updateUserCredential(ucList[0].Id, ucList[0].Status__c);
```
Comment (nitinsavanurness) [inline force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls]: asserts are missing

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.css [hunk 6]
```diff
@@ -0,0 +1,10 @@
+.imageAlign
+{
+float: left;
+font-weight: 700;
+}
+.credAssign{
+    font-weight: bold;
+    padding-top:14px;
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.css]: change px to rem

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+                                    <thead>
+                                    </thead> 
+                                    <tbody>
+                                        <template for:each={userCredentialsData} for:item="u">  
+                                            <tr class="slds-hint-parent" key = {u.Id} >
+                                                <th></th>
+                                                <td data-label="Credentials">                                                    
+                                                    <lightning-layout vertical-align="center" horizontal-align="left" >
+                                                        <lightning-layout-item flexibility="auto" size="3" padding="horizontal-Small">
+                                                            <div class="slds-box slds-align_absolute-center" style="border:0px;float:left;margin-left:-33px;padding-bottom:0px">
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html]: move all the inline styling to css class

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+                            <div class="slds-scrollable_y"  style={boxStyle}>
+                                <table class="slds-table " aria-describedby="To display User's Active Credentils">
+                                    <thead>
+                                    </thead> 
+                                    <tbody>
+                                        <template for:each={userCredentialsData} for:item="u">  
+                                            <tr class="slds-hint-parent" key = {u.Id} >
+                                                <th></th>
+                                                <td data-label="Credentials">                                                    
+                                                    <lightning-layout vertical-align="center" horizontal-align="left" >
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html]: define device size as well

**Review by nitinsavanurness — COMMENTED** [review]
@lavanyayezzuness  - please see the comments inline

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls (hunk 1 above)
Comment (lavanyayezzuness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: Changes updated please review

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls (hunk 2 above)
Comment (lavanyayezzuness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: Changes updated  please review

File: force-app/main/default/classes/CredentialTrackingLWCAPEX.cls (hunk 3 above)
Comment (lavanyayezzuness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX.cls]: Changes updated please review

File: force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls (hunk 4 above)
Comment (lavanyayezzuness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls]: Changes Updated please review

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.css (hunk 6 above)
Comment (lavanyayezzuness) [reply force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.css]: Changes Updated please review

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html (hunk 8 above)
Comment (lavanyayezzuness) [reply force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html]: Changes updated please review

File: force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html (hunk 7 above)
Comment (lavanyayezzuness) [reply force-app/main/default/lwc/credentialTrackingLWC/credentialTrackingLWC.html]: Changes updated  please review

File: force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls (hunk 5 above)
Comment (lavanyayezzuness) [reply force-app/main/default/classes/CredentialTrackingLWCAPEX_Test.cls]: Changes Updated please Review

**Review by nitinsavanurness — APPROVED** [review]

**Pull request merged**